cmplibrary/src/main/java/com/sourcepoint/gdpr_cmplibrary/ConsentAction.java

```diff
@@ -26,17 +26,11 @@
         this(actionType, choiceId, null, requestFromPm, pmSaveAndExitVariables);
     }
 
-    public static ConsentAction getEmptyDismissAction(boolean isPmOn) {
```

Comment:
Was this method deleted because it was unused?

Reply:
@andresilveirah Yes this method was unused.